addons/web/static/src/views/form/form_status_indicator/form_status_indicator.js

```diff
@@ -32,4 +32,5 @@ FormStatusIndicator.props = {
     save: Function,
     discard: Function,
     isDisabled: Boolean,
+    fieldIsDirty: Boolean,
```

Comment:
this is a larger ref, maybe for master if we decided to do it, but I'm wondering if it's a good idea that this widget receives the model in props. What if it directly receives the status ("invalid", "dirty" or "saved")?

Reply:
It's a good idea, we only use the model for the status.